int16 CCT
int16 intensity
int16 x
int16 y
---
string cmdstr